### ISSUE TITLE
Black square being read out rather than bullet in validation error message 

### ISSUE THREAD

**Issue by msinga** [Sage/carbon#6440]
### Desired behaviour

Something I have  noticed and heard when we use VoiceOver to read out the validation listed items - the VoiceOver tool (tested on Mac) reads out "black square"  then the text

Should read out bullet 

### Current behaviour

Due to the List carbon component, rather than something you're doing explicitly in the code. The square also gets read out on carbon storybook when using an unordered list. 

And in the carbon code, the ui gets styled with the black square.

### Suggested Solution

It should say 'bullet'. Sounds like it might be an issue with the list mark-up that's being used 

### CodeSandbox or Storybook URL

https://carbon.sage.com/?path=/docs/typography--variants-story

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by robinzigmond** [comment]
@tempertemper this seems to be because we have `list-style-type: square` in the CSS for this list. If we remove that Voiceover says "bullet" instead of "black square". I'm not sure this is a real issue as that seems more accurate when it's a square rather than a bullet?

But it's easy to remove that CSS if we think that's better - @ljemmo what are your thoughts on this?

**Comment by robinzigmond** [comment]
closing based on the following responses on Slack from Chris Merrington:

It is caused by the CSS and screen reader users will have likely come across it before. It isn’t a WCAG fail but could be annoying for some users.
The most common way around this I have seen is to remove styling from the list, add pseudo elements as a replacement for the bullet points and then hide these pseudo elements using aria-hidden.


Spoke to [@Martin Underhill] about this. We agreed that is likely only a small percentage of screen reader users who would find it annoying and the effort needed to sort this likely outweighs the need for it.
We are happy for you to leave it as it is and close the ticket
Thanks for bringing it up though [@Mark]. It is something we can pick up at a later date.
